csrc/fusion.h

```diff
@@ -242,21 +247,51 @@ class TORCH_CUDA_CU_API Fusion : public IrContainer {
     return io_alias_;
   }
 
-  // vector of (tv, dim, selection)
-  // For each entry in the vector, the selected tv/expr in loop tv->axis(dim)
-  // will be rotated
-  using LoopRotationParam = std::vector<
-      std::tuple<TensorView*, int64_t, std::unordered_set<Statement*>>>;
+  // Fusion-managed data is a mechanism to communicate data that survives fusion
+  // clone.
+  using CloneFn = std::function<std::any(IrCloner&, std::any)>;
 
-  const LoopRotationParam& getLoopRotationParam() const {
-    return loop_rotation_param_;
+  inline size_t manage(std::any data, CloneFn clone) {
```

Comment:
What is this `manage` supposed to do? Can you please add comments?

Reply:
I added detailed doc and a test `FusionManagedData_CUDA` to explain this.